### DISCUSSION TITLE
不能单个调控程序的音量

### DISCUSSION THREAD

**Discussion by zeminn** [File-New-Project/EarTrumpet#1331] [Q&A] [answered]
在打开Dolby Access设置音效之后，点开音量图标后，只有主音量调节，程序音量调节消失了

**Comment by riverar** [comment, marked as answer]
@JerseyMany-23 Hi, this is a known issue https://github.com/File-New-Project/EarTrumpet/issues/1305 thanks!

您好，这是一个已知问题 https://github.com/File-New-Project/EarTrumpet/issues/1305 谢谢！
